server/src/main/java/org/apache/cloudstack/outofbandmanagement/PowerOperationTask.java

```diff
@@ -17,16 +17,26 @@
 
 package org.apache.cloudstack.outofbandmanagement;
 
+import com.cloud.event.ActionEventUtils;
+import com.cloud.event.EventTypes;
+import com.cloud.event.EventVO;
 import com.cloud.host.Host;
+import org.apache.cloudstack.context.CallContext;
+import org.apache.cloudstack.framework.config.ConfigKey;
+import org.apache.cloudstack.framework.config.Configurable;
 import org.apache.log4j.Logger;
 
-public class PowerOperationTask implements Runnable {
+public class PowerOperationTask implements Runnable, Configurable {
     public static final Logger LOG = Logger.getLogger(PowerOperationTask.class);
 
     final private OutOfBandManagementService service;
     final private Host host;
     final private OutOfBandManagement.PowerOperation powerOperation;
 
+    public static final ConfigKey<Boolean> CREATE_EVENT_ON_OOBM_FAILURE = new ConfigKey<Boolean>("Advanced", Boolean.class, "create.event.on.oob.failure", "true",
```

Comment:
I know there is no naming scheme here, but isn't it easier if OOB related config keys are prefixed with *oob*?

<pre>oob.create.event.on.failure</pre>

For example

Reply:
Thanks for the suggestion, @wido.
I am removing that global settings config key. There are too many config keys already, I am leaving to add a new one only if it is really necessary.